apps/api/src/storage/test/storage.acceptance.spec.ts

```diff
@@ -591,14 +594,13 @@ describe("Storage Acceptance", () => {
         }
       ];
 
-      await req
-        .post("/storage", serialize({content: objects}), {
-          "Content-Type": "application/bson"
-        })
-        .catch(error => {
-          expect(error.response.statusCode).toBe(400);
-          expect(error.response.message).toBe("An object with this name already exists.");
-        });
+      const res = await req.post("/storage", serialize({content: objects}), {
+        "Content-Type": "application/bson"
+      });
+
+      expect(res.statusCode).toEqual(400);
+      expect(res.statusText).toEqual("Bad Request");
+      expect(res.body.message).toEqual("An object with this name already exists.");
```

Comment:
Tests were silently passing before, since bad request exceptions never caused promise rejection. This way we expect the bad request exception more explicitly and tests will fail if storage accepts inserting two objects with identical names.